nlp_profiler/noun_phase_count.py

```diff
@@ -0,0 +1,21 @@
+import en_core_web_sm
+import spacy
+from nlp_profiler.constants import NaN
+nlp = en_core_web_sm.load()
+
+
+def gather_nouns(sent: str):
```

Comment:
minor: good to have full names `sentence` rather than partials, helps readability,